cornice/cors.py

```diff
@@ -125,7 +125,7 @@ def apply_cors_post_request(service, request, response):
             'Access-Control-Allow-Credentials' not in response.headers):
         response.headers['Access-Control-Allow-Credentials'] = 'true'
 
-    if request.method is not 'OPTIONS':
+    if request.method != 'OPTIONS':
```

Comment:
On python 2.7.5+, I don't have the problem when turning it back and running the test, with a `pdb.set_trace()`:

```
> /home/alexis/dev/github.com/cornice/cornice/cors.py(129)apply_cors_post_request()
-> if request.method != 'OPTIONS':
(Pdb) request.method != 'OPTIONS'
False
(Pdb) request.method is not 'OPTIONS'
False
```